projects/plugins/jetpack/modules/custom-post-types/portfolios.php

```diff
@@ -555,39 +667,44 @@ function allow_portfolio_rest_api_type( $post_types ) {
 	 * Our [portfolio] shortcode.
 	 * Prints Portfolio data styled to look good on *any* theme.
 	 *
+	 * @param array $atts Shortcode attributes.
+	 *
 	 * @return portfolio_shortcode_html
 	 */
-	static function portfolio_shortcode( $atts ) {
-		// Default attributes
-		$atts = shortcode_atts( array(
-			'display_types'   => true,
-			'display_tags'    => true,
-			'display_content' => true,
-			'display_author'  => false,
-			'show_filter'     => false,
-			'include_type'    => false,
-			'include_tag'     => false,
-			'columns'         => 2,
-			'showposts'       => -1,
-			'order'           => 'asc',
-			'orderby'         => 'date',
-		), $atts, 'portfolio' );
-
-		// A little sanitization
-		if ( $atts['display_types'] && 'true' != $atts['display_types'] ) {
-			$atts['display_types'] = false;
-		}
-
-		if ( $atts['display_tags'] && 'true' != $atts['display_tags'] ) {
-			$atts['display_tags'] = false;
-		}
+	public static function portfolio_shortcode( $atts ) {
+		// Default attributes.
+		$atts = shortcode_atts(
+			array(
+				'display_types'   => true,
+				'display_tags'    => true,
+				'display_content' => true, // Can be false, true, or full.
```

Comment:
I found this a bit confusing when working on those attributes, so had to add a comment here.